src/main/java/com/arias/kadai6/HelloController.java

```diff
@@ -0,0 +1,44 @@
+package com.arias.kadai6;
+
+
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+
+    @Async
+    @GetMapping("/hello")
+    public String exampleAsyncEndpoint() {
+
+        //クラスインスタンスの生成
+        RunWaiting1 runWaiting1 = new RunWaiting1();
+        RunWaiting2 runWaiting2 = new RunWaiting2();
+        RunWaiting3 runWaiting3 = new RunWaiting3();
+
+        //スレッドの生成
```

Comment:
スレッドの生成と待機は別メソッドに切り出してもよいかもしれませんね！
```suggestion
        //スレッドの生成
       private Thread createAndStartThread(Runnable runnable) {
        Thread thread = new Thread(runnable);
        thread.start();
        return thread;
    }

    private void waitForThreadsToComplete(Thread... threads) {
        try {
            for (Thread thread : threads) {
                thread.join();
            }
        } catch (InterruptedException e) {
            Thread.currentThread().interrupt();
            throw new RuntimeException(e);
        }
    }
```